src/ggrc-client/js/models/conflict-resolution/conflict-resolution.js

```diff
@@ -42,30 +42,30 @@ export function tryResolveConflictValues(baseAttrs, attrs, remoteAttrs, obj) {
   return hasConflict;
 }
 
-export default function resolveConflict(xhr, obj) {
+export default function resolveConflict(xhr, obj, remoteAttrs) {
   let attrs = _.merge({}, obj.attr());
   let baseAttrs = _.merge({}, obj._backupStore()) || {};
 
-  return obj.refresh().then(function (obj) {
-    let stillHasConflict = false;
-    let remoteAttrs = _.merge({}, obj.attr());
+  // merge current instance with remote attributes
+  obj.attr(remoteAttrs);
 
-    if (_.isEqual(remoteAttrs, attrs)) {
-      // current state is same as server state -- do nothing.
-      return obj;
-    } else if (_.isEqual(remoteAttrs, baseAttrs)) {
-      // base state matches server state -- no incorrect expectations -- save.
-      return obj.attr(attrs).save();
-    }
-    // check what properties changed -- we can merge if the same prop wasn't changed on both
-    stillHasConflict =
-      tryResolveConflictValues(baseAttrs, attrs, remoteAttrs, obj);
-    if (stillHasConflict) {
-      notifierXHR('warning', {status: 409});
-      xhr.remoteObject = remoteAttrs;
-      return new $.Deferred().reject(xhr, 409, 'CONFLICT');
-    }
+  let stillHasConflict = false;
 
-    return obj.save();
-  });
+  if (_.isEqual(remoteAttrs, attrs)) {
+    // current state is same as server state -- do nothing.
+    return obj;
+  } else if (_.isEqual(remoteAttrs, baseAttrs)) {
+    // base state matches server state -- no incorrect expectations -- save.
+    return obj.attr(attrs).save();
```

Comment:
```suggestion
    return obj.save();
```
Also, no need to apply local attributes to local object before saving.